core/peerset/src/lib.rs

```diff
@@ -285,6 +287,11 @@ impl PeersetMut {
 		}
 		alloc_slots(&mut inner, &self.parent.tx);
 	}
+
+	/// Produces a JSON object containing the state of the peerset manager, for debugging purposes.
+	pub fn debug_info(&self) -> serde_json::Value {
+		serde_json::Value::Null
```

Comment:
Debug nothing?

Reply:
Yeah, the `peerset` crate is a stub that will be implemented in the near future.